### DISCUSSION TITLE
Not able to generate a XML feed for Google

### DISCUSSION THREAD

**Discussion by Valenz1** [bullhorn/career-portal#437] [Q&A]
I am trying to generate an XML feed for a site map on Google based on the information here https://bullhorn.github.io/career-portal/search-engine-indexing.html 

When I go to enter the information [your career portal root]/feed I am not able to generate a feed. Can anyone provide some insight on what I might be doing wrong? Thanks!

**Comment by charlesabarnes** [comment]
Hello, this is only available when using the "advanced hosting" option. This version comes with a node backend that generates the feed for you, along with some other features for server side rendering

**Comment by Valenz1** [comment]
We are using the advanced hosting option. Could there be anything else causing it?

Greg Valenzuela
Manager of Back Office Operations
Tel. 909-417-2827
OfficeWorksRx ... Your Premier Search Consultant
visit us online at https://officeworksrx.com/
[megaphone]Help Get Someone a Job and Receive a $25 Amazon Giftcard!<https://officeworksrx.com/referral-program>
Adaptability | Includer | Context | Deliberative | Futuristic

**Reply by charlesabarnes** [reply]
Can you provide a link to your portal so I can take a look?

**Comment by Valenz1** [comment]
Yes of course. It is https://officeworksrx.com/jobs/#/

**Reply by charlesabarnes** [reply]
Hello,

This appears as if it's not the advanced hosting option.   It is the statically hosted option (one that you upload with FTP or some similar hosting solution).  The advanced option is one where you run and host a node server that serves a server side rendered version of the career portal. That version is required for the XML features.

**Reply by davidfchuck** [reply]
Hi Charles,
I'm running the advanced hosting option... but I'm not getting a sitemap page that's in XML format.  It's generating a sitemap, but it's only in HTML format... so Google Console errors it out.  Same with RSS feed.

my site is  "jobs.verticalrecruiting.com"

Thanks in advance for any assistance!

**Comment by pbearne** [comment]
Hi David,

As you are using WordPress look at Matadorjobs.com as our plugin has a custom XML feeds add-on. 
You get the site map for free from WP.
You can move the job board into the primary domain and the SEO benefits.

Paul@matadorjobs.com
